### ISSUE TITLE
ton+eth increase efficiency

### ISSUE THREAD

**Issue by karmagkz** [Lolliedieb/lolMiner-releases#947]
### Discussed in https://github.com/Lolliedieb/lolMiner-releases/discussions/945

<div type='discussions-op-text'>

<sup>Originally posted by **karmagkz** January 10, 2022</sup>
1,42v in the current situation when starting ton + eth mining with 2 .bat
ton efficiency drops by almost 3 times
eth efficiency drops by 2 times

is it possible to distribute the load between gpu and memory for two different coins to increase efficiency?</div>

**Comment by megalolka** [comment]
Yes, could you make a dual mining ETH + TON by one instance of lolminer?

**Comment by karmagkz** [comment]
I think @Lolliedieb can you help us with the settings
